src/sentry/static/sentry/app/stores/globalSelectionStore.jsx

```diff
@@ -119,10 +119,7 @@ const GlobalSelectionStore = Reflux.createStore({
 
     this.selection = {
       ...this.selection,
-      datetime: {
-        ...this.selection.datetime,
-        ...datetime,
-      },
+      datetime,
```

Comment:
There was a slight bug when switching from relative to absolute would preserve the relative value before then being updated again with the correct `relative: null`.